### PR TITLE
f-content-cards@v4.7.0 🔟 Add 10% stampcards graphics for UK rollout

### PR DESCRIPTION
**Pull request by minrwhite**
- Added 10% stampcard graphics/Fixed 15% stampcard graphics for positioning
- Added stampcard percentage config to AU and UK

---

## UI Review Checks

Screenshots from storybook:

### 15% (AU standard):
image

### 10% (UK standard):
image

- [x] README and/or UI Documentation has been [created|updated]
- [x] Unit tests have been [created|updated]
- [ ] ~This code has been checked with regard to [our accessibility standards](http://fozzie.just-eat.com/documentation/general/accessibility/checklist)~ No change to accessiblity

## Browsers Tested

- [x] Chrome (latest)
- [x] Firefox (latest)
- [ ] Internet Explorer 11
- [ ] Mobile (Please list device/browser – Ideally one iPhone model and one Android)

### List any other browsers that this PR has been tested in:

- [View the Browser Support Checklist](http://fozzie.just-eat.com/documentation/general/browser-support)

### CODE REVIEW

File: packages/components/organisms/f-content-cards/src/components/cardTemplates/images/stamp-empty-10.svg [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="48" height="48">
```
Comment (minrwhite) [inline packages/components/organisms/f-content-cards/src/components/cardTemplates/images/stamp-empty-10.svg:1]: These are run through SVGOMG - though with pretty-print enabled, as indentation/newlines only add about 20 bytes.

File: packages/components/organisms/f-content-cards/src/components/cardTemplates/_tests/StampCard1.test.js [hunk 2]
```diff
@@ -40,7 +41,19 @@ function getWrapper (cardDetails = {}, testId = 'stampCard1') {
         directives: {
             makeTextAccessible: jest.fn()
         },
-        provide
+        stubs: [
+            'CardCase',
+            'EmptyStamp15',
+            'EmptyStamp10',
+            'FullStamp15',
+            'FullStamp10'
```
Comment (minrwhite) [inline packages/components/organisms/f-content-cards/src/components/cardTemplates/_tests/StampCard1.test.js:49]: These are required to allow the stubs to be locatable by `findComponent` below.

File: packages/components/organisms/f-content-cards/src/components/cardTemplates/StampCard1.vue [hunk 3]
```diff
@@ -81,17 +82,22 @@ import format from 'date-fns/format';
...
+    '15'
+];
+
 export default {
     name: 'StampCard1',
 
     components: {
-        EmptyStamp,
-        FullStamp,
+        EmptyStamp15: () => import('./images/stamp-empty-15.svg'),
```
Comment (matthewhardern) [inline packages/components/organisms/f-content-cards/src/components/cardTemplates/StampCard1.vue:97]: Not thought of doing this, cool idea

**Review by matthewhardern — APPROVED** [review]

**Comment by AlixRichards** [comment]
I have a small question for my understanding - 
The UI screenshots from story book added above show AU and UK.
Both have a line of text `You've accumulated £6.83 so far` -- Does it matter that this is not a $ sign for AU?

**Comment by minrwhite** [comment]
Thanks, @AlixRichards - this message, including the currency symbol used are included in the subtitle as passed from the content cards backend [as per here](https://github.com/justeat/fozzie-components/blob/029bdb4edf5533945155e831fa725b0b0ba3c8ce/packages/components/organisms/f-content-cards/stories/StampCard.stories.js#L103) and so are under the control of CRM rather than the localisation of this component specifically.

**Comment by AlixRichards** [comment]
Ok perfect, that makes sense, thank you @benwhite-deltas 👍

**Review by AlixRichards — APPROVED** [review]

**Pull request merged**